packages/merge-styles/tslint.json

```diff
@@ -1,4 +1,6 @@
 {
   "extends": ["office-ui-fabric-react-tslint"],
-  "rules": {}
+  "rules": {
+    "no-any": false
```

Comment:
Is `// tslint:disable-next-line:no-any` usage insufficient?

Reply:
It is very noisy to litter this all over the place. There are alot of legitimate uses of any here in this repo because of insufficient expressiveness of the TS typings atm.